yaks/spec/unit/yaks/default_policy/derive_mapper_from_object_spec.rb

```diff
@@ -3,16 +3,79 @@
 
   let(:options) { {} }
 
+  it 'should use const_get with second argument set to false' do
+    expect(Object).to receive(:const_get).with(kind_of(String), false)
```

Comment:
alright. If you can get it to work with the exact string it will get that would be even better. But `any_args` should be fine. Mutant will tell you if you missed anything :)